kyuubi-server/src/main/java/org/apache/kyuubi/session/FileSessionConfAdvisor.java

```diff
@@ -0,0 +1,43 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one or more
+ * contributor license agreements.  See the NOTICE file distributed with
+ * this work for additional information regarding copyright ownership.
+ * The ASF licenses this file to You under the Apache License, Version 2.0
+ * (the "License"); you may not use this file except in compliance with
+ * the License.  You may obtain a copy of the License at
+ *
+ *    http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.apache.kyuubi.session;
+
+import org.apache.kyuubi.Utils;
+import org.apache.kyuubi.config.KyuubiConf;
+import org.apache.kyuubi.plugin.SessionConfAdvisor;
+
+import java.io.File;
+import java.util.HashMap;
+import java.util.Map;
+
+public class FileSessionConfAdvisor implements SessionConfAdvisor {
```

Comment:
better to write in scala to keep consistent

Reply:
done